cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/model/proofs/CommitmentWellformednessCommonInput.java

```diff
@@ -0,0 +1,37 @@
+package org.cryptimeleon.incentivesystem.cryptoprotocol.model.proofs;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+import org.cryptimeleon.craco.protocols.CommonInput;
+import org.cryptimeleon.incentivesystem.cryptoprotocol.model.IncentivePublicParameters;
+import org.cryptimeleon.math.serialization.Representable;
+import org.cryptimeleon.math.serialization.Representation;
+import org.cryptimeleon.math.serialization.annotations.ReprUtil;
+import org.cryptimeleon.math.serialization.annotations.Represented;
+import org.cryptimeleon.math.structures.groups.GroupElement;
+
+@Value
+@AllArgsConstructor
+public class CommitmentWellformednessCommonInput implements CommonInput, Representable {
```

Comment:
Why is this representable? I think the values can be sent in the Request itself

Reply:
This is a relic from the ancient times when I thought that it was a good idea to serialize the CommonInput.
Now we just recompute it if needed somewhere else than where it originated, so it is never actually serialized.
I removed Representable from the implements-clause.